README.md

```diff
@@ -14,41 +14,36 @@
+For even more advanced features, consider the Supporter Edition (SE). The SE version offers additional functionality to enhance your experience. Learn more about the Supporter Edition and its benefits [here](https://lycheeorg.github.io/get-supporter-edition).
 
-## Notice: `master` & `alpha` branches
-
-As LycheeOrg is a very small team, we do not have many maintainers. Most of us have an active work/family life, and as a result, it is no longer possible for us to apply proper 4-eyes principle in the coding reviews.
+## Support the Team
 
-In order to keep our high code quality, the following changes have been applied.
-
-- `master` stays as a stable branch and contains 4-eyes peer-reviewed pull-requests.
-- `alpha` contains the latest changes (i.e. the above mentionned PR) merged with minimal review.
+We aim to maintain a free open-source photography library with high quality of code.<br>
+Being in control of our own data, our own pictures is something that we value above all.
 
-With this change, we hope to strike a balance between decently paced development (on `alpha`) and maintaining a robust core (on `master`).
+Through [contributions, donations, and sponsorship](https://github.com/sponsors/LycheeOrg), you allow Lychee to thrive. Your donations directly support demo server costs, continuous enhancements, and most importantly bug fixes!
 
-On Docker, `nightly`/`dev` continues to refer to the latest `master` commit.
-The `alpha` tag is updated daily with the content of the associated branch.
+## Installation
 
-That being said, if you like the gallery and would like to contribute, do not hesitate to open pull request. If you would like to see more functionalities added and help us push Lychee, [Join the team!](https://lycheeorg.github.io/docs/contributions.html#joining-the-team)
+There are three deployment options available. The simplest is **Docker deployment**, as all dependencies are already predefined and configured.
 
-### Support the Team
+### File-based deployment
 
-We aim to maintain a free open-source photography library with high quality of code.<br>
-Being in control of our own data, our own pictures is something that we value above all.
+Copy the extracted Zip file from https://github.com/LycheeOrg/Lychee/releases to your webserver
 
-Through [contributions, donations, and sponsorship](https://github.com/sponsors/LycheeOrg), you allow Lychee to thrive. Your donations directly support demo server costs, continuous enhancements, and most importantly bug fixes!
+### Docker deployment
 
+Alternatively, an official Docker image can be found at [LycheeOrg/Lychee-Docker](https://github.com/LycheeOrg/Lychee-Docker) or on Docker Hub as [lycheeorg/lychee](https://hub.docker.com/r/lycheeorg/lychee).
```

Comment:
This should probably be above File-based if we're recommending it.

Reply:
File-based has the disadvantage that many dependencies such as ffmpeg or the php version must fit. This catalog of dependencies does not exist with docker and the installation of the lychee docker variant requires 3 commandos. So maybe the way should point in that direction it is definitely the way I would recommend to install it today.

Reply:
Agreed, it just feels weird saying "We recommend using Docker, but first, here's how to do it a different way" :)

Reply:
I turned it